docs/OV_Runtime_UG/OpenVINO_Runtime_User_Guide.md

```diff
@@ -7,27 +7,22 @@
 .. toctree::
    :maxdepth: 1
    :hidden:
-   
+
    openvino_2_0_transition_guide
    openvino_docs_IE_DG_Integrate_with_customer_application_new_API
    openvino_docs_OV_Runtime_UG_Model_Representation
    ngraph_transformation
    openvino_docs_deployment_optimization_guide_dldt_optimization_guide
    openvino_docs_IE_DG_Device_Plugins
-   Direct ONNX Format Support <openvino_docs_IE_DG_ONNX_Support>
-   openvino_docs_IE_DG_Paddle_Support
    openvino_docs_IE_DG_Int8Inference
    openvino_docs_IE_DG_Bfloat16Inference
    openvino_docs_IE_DG_DynamicBatching
    openvino_docs_IE_DG_ShapeInference
    openvino_docs_IE_DG_Model_caching_overview
    openvino_docs_IE_DG_Extensibility_DG_Intro
-   openvino_docs_IE_DG_Memory_primitives
    openvino_docs_IE_DG_network_state_intro   
-   openvino_docs_IE_DG_API_Changes
-   openvino_docs_IE_DG_Known_Issues_Limitations
-   openvino_docs_IE_DG_Glossary
-      
+   openvino_docs_OV_Runtime_API_Changes
```

Comment:
looks like you removed this code [docs/OV_Runtime_UG/API_Changes.md](https://github.com/openvinotoolkit/openvino/pull/10504/files#diff-2d78cb8b15efd4a85328cb435eb4fe8f96269da8c608a57acf3a7700306e9e79)

@andrew-zaytsev could you please advice where we put this to resources?

Reply:
No it was changed to:
<img width="2954" alt="Screenshot 2022-02-18 at 11 20 50" src="https://user-images.githubusercontent.com/4737252/154645003-51b1b87f-6b18-4b4a-859a-341938a1ea11.png">